content/new-in-22.md

```diff
@@ -199,40 +189,51 @@ Provide ways to help users navigate, find content, and determine where they are.
 </blockquote>
 <p><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum">Understanding Focus Not Obscured (Minimum)</a></p>
 
-### 2.4.12 Focus Not Obscured (Enhanced) (AAA)
-<p><em>(Persona, problem, and works well<!--, and "in brief" --> same as 2.4.12 above.)</em></p>
+### [EOWG draft] 2.4.12 Focus Not Obscured (Enhanced) (AAA)
+
+<p>In brief:</p>
+<ul>
+      <li><p><span class="whatwhy">What to do: </span>Ensure that when items have focus, they are fully visible.</p></li>
```

Comment:
```suggestion
      <li><p><span class="whatwhy">What to do: </span>Ensure no part of the item receiving keyboard focus is hidden.</p></li>
```

Reply:
FYI in WCAG it currently reads "Ensure no content obstructs the item receiving keyboard focus"
If there's any interest in putting the emphasis on blocking content, could always make this a bit more gracefully phrased as "Ensure content does not obstruct the item receiving keyboard focus" or something of that nature ("does not cover", "does not block", etc)

Reply:
Seems like it would be good to have similar wording for minimum and enhanced.

Reply:
To clarify, the last two are variations on the same one, in case that wasn't clear. So the pair would be:
> Ensure the item getting keyboard focus is visible
 Ensure the item getting keyboard focus is fully visible

But I think I agree that the difference may not be enough